modules/theme/src/themes/default/elements/label.overrides

```diff
@@ -69,6 +69,9 @@
             }
         }
     }
+    .detail {
```

Comment:
This is a global change, every `detail` class in a label would get this change. Is this expected @Rashmini ?

Reply:
Fixed with 01eb52b63cc50bfaf6005c2da9e71ec88b0c3854